src/components/gamespage/GamesPageImageGrid/index.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import '../../../matchMedia';
+import renderer from 'react-test-renderer';
+import GamesPageImageGrid from '.';
+
+it('GamesPageImageGrid renders correctly', () => {
+    const tree = renderer.create(<GamesPageImageGrid />).toJSON();

```

Comment:
i'm not gonna say what's wrong because i want you to figure it out but i want you to look at this line and think about why somtehing might be undefined

and think about whether test code belongs only in test.js files or in GamesPage/index.jsx

Reply:
oh god :D I see it

Reply:
I was about to implement some dependency changes lol

Reply:
PTAL. To keep data same everywhere, I exported data from container and imported it from test. If there is a better way to do it, please let me know.